helm/cluster-api-provider-azure/templates/apps_v1_deployment_capz-controller-manager.yaml

```diff
@@ -64,34 +68,39 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: "{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
-        imagePullPolicy: Always
+        image: '{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}'
+        imagePullPolicy: IfNotPresent
```

Comment:
Why change from Always here?

Reply:
No particular reason, it's just what upstream sets. If you think it's useful having `Always` I can change it back.

Reply:
Works for me if upstream sets it like this.